ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/views/BelphegorView.ui.xml

```diff
@@ -48,6 +48,10 @@
              type="com.sencha.gxt.widget.core.client.TabItemConfig">
         <ui:attributes text="Metadata Templates"/>
     </ui:with>
+    <ui:with field="toolAdminConfig"
+             type="com.sencha.gxt.widget.core.client.TabItemConfig">
+        <ui:attributes text="Tool Admin"/>
+    </ui:with>
```

Comment:
I suggest putting this tab next to the "Tool Requests" tab, rather than just at the end.